src/cellfunction/cellfunction.jl

```diff
@@ -519,53 +454,31 @@ function LazyOperators.materialize(
     sideFacePoint::Side⁻{Nothing, <:Tuple{<:FacePoint}},
 )
     cnodes_n, cnodes_p, ctype_n, ctype_p, ξ_n, ξ_p = _unpack_face_point(sideFacePoint)
-    return _coplanar_rotation(cnodes_n, cnodes_p, ctype_n, ctype_p, ξ_n, ξ_p)
+
+    # We choose to use `cell_normal` instead of `normal`,
+    # but we could do the same with the latter.
+    ν_n = cell_normal(ctype_n, cnodes_n, ξ_n)
+    ν_p = cell_normal(ctype_p, cnodes_p, ξ_p)
+
+    return _coplanar_rotation(ν_n, ν_p)
 end
 
 function LazyOperators.materialize(
-    ::CoplanarRotation,
+    cr::CoplanarRotation,
     sideFacePoint::Side⁺{Nothing, <:Tuple{<:FacePoint}},
 )
-    cnodes_n, cnodes_p, ctype_n, ctype_p, ξ_n, ξ_p = _unpack_face_point(sideFacePoint)
-    return _coplanar_rotation(cnodes_p, cnodes_n, ctype_p, ctype_n, ξ_p, ξ_n)
+    fPoint, = get_args(sideFacePoint)
+    materialize(cr, Side⁻(opposite_side(fPoint)))
```

Comment:
the necessity of `opposite_side` is just for dispatch problem write? Not for numerical reason?